api-services/api.service.js

```diff
@@ -79,32 +91,14 @@ export class ApiService {
         throw exception
     }
 
-    logActionRequest (actionMessage, requestConfig) {
-        const httpMethod = requestConfig.method
-        const url = requestConfig.url
-        const logMessage = buildBasicLogMessage(extractCorrelationId(requestConfig))
-
-        logMessage.message = actionMessage
-        logMessage.apiServiceUrl = buildLoggingUrl(httpMethod, url)
-
-        // We only want to log health check endpoints as debug
-        if (isHealthUrl(url)) {
-            this.logger.debug(logMessage)
-        } else {
-            this.logger.info(logMessage)
-        }
-    }
-
     logApiServiceException (requestConfig, exception) {
-        const httpMethod = requestConfig.method
-        const url = requestConfig.url
-
-        const logMessage = buildBasicLogMessage(extractCorrelationId(requestConfig))
-        logMessage.message = 'ApiService Exception'
-        logMessage.apiServiceUrl = buildLoggingUrl(httpMethod, url)
-        logMessage.errorStatus = exception.response ? exception.response.status : 'No Response'
-        logMessage.exception = exception
-
-        this.logger.error(logMessage)
+        const { correlationId, httpMethod, url } = extractLogMessageInfo(requestConfig)
+        const actionMessage = 'ApiService Exception'
+        // We may want to decide on a consistent error format, but confirmed with Paul a properties object works
+        const properties = {}
+        properties.errorStatus = exception.response ? exception.response.status : 'No Response'
+        properties.exception = exception
```

Comment:
This could just be declared in one go instead of instantiating an empty object and adding a property to it

```
 const properties = {
  errorStatus: exception.response ? exception.response.status : 'No Response',
  exception
}
```

Reply:
Agreed...